goalgorilla/open_social
# Issue #3164311 by rolki: When trying to save "admin/appearance/settings/gin" I get an error

## Problem
When trying to save "admin/appearance/settings/gin" I get an error

## Solution
Add an extra condition to "improved_theme_settings_form_system_theme_settings_alter"

## Issue tracker
* https://www.drupal.org/project/social/issues/3164311 
* https://getopensocial.atlassian.net/browse/YANG-3508 

## How to test
1. Do a complete new install
2. Observe that Gin is set as the admin theme
3. Go to the "/admin/appearance/settings/gin"
4. Click "Save Configuration".
